working-with-openai/setting-up-open-ai-postman.md

```diff
@@ -0,0 +1,91 @@
+We will talk more in the next lesson about what this request is and how it works, but for now, we will use it to verify that our environment is set up correctly. If you try and make the request now, you will get the following response body with an error that states you are missing an API key.  
+![OpenAI POST Response with Missing API Key Error](assets/setting-up-openai-postman-and-java/open-ai-postman-missing-api-key-error.png)  
+*Fig. OpenAI Postman response body with Missing API Key Error*
+
+3. Let's go ahead and add our authorization now. Open up the Authorization tab in Postman and you should see a pop up that asks if you need help authorizing the OpenAI API. Click "Set Up New Authorization"  
+![Postman Authorization Tab with New Authorization circled](assets/setting-up-openai-postman-and-java/open-ai-postman-set-up-new-authorization.png)  
+*Fig. OpenAI authorization tab with "Set Up New Authorization" circled*  
+
+4. You will be taken to a window to enter your OpenAI API key. Enter your API key and press "Save authorization". 
+![Tab to enter API key in Postman](assets/setting-up-openai-postman-and-java/open-ai-postman-enter-api-key.png)  
+*Fig. Tab to enter API key in Postman*
+
+5. Now let's check to see if our API key is registered! If you send the request now, you will see the following error.  
+![Open AI Postman Model Parameters Needed Error](assets/setting-up-openai-postman-and-java/open-ai-postman-model-parameters-needed-error.png)  
+*Fig. Model parameters needed error*  
+
+If you get this error, it's a pretty sure sign your API key is working. However, let's add a request body to make sure we are able to get a working response.  Go ahead and navigate to the body tab and select "raw" with JSON as the format. Copy and paste the following request body into the window:
+
+```py
+{
+    "model" : "gpt-3.5-turbo",
+    "messages" : [
+        {
+            "role" : "user",
+            "content" : "Write me a list of 5 flightless birds."
+        }
+    ]
+}
+```
+You should now be able to submit your POST request and get a response more in line with what you are looking for. Your response may look slightly different, but a sample of what you could see is shown below:  
+
+![Successful API response](assets/setting-up-openai-postman-and-java/open-ai-postman-response-success.png) 
+*Fig. Successful OpenAI POST request with response body ([Full size image](assets/setting-up-openai-postman-and-java/open-ai-postman-response-success.png))*  
+
+If you've made it this far, congratulations! You now have Postman configured to practice the openAI requests that we will be going over in the next couple of lessons. If you saved your environment correctly, it should now exist as an option in Postman when you choose your environment. If for any reason your requests still aren't working, double check your work or reach out to an instructor for further assistance! 
```

Comment:
To encourage folks to reach out to the wider community vs directly to instructors, we could tweak the last sentence.

```suggestion
If you've made it this far, congratulations! You now have Postman configured to practice the openAI requests that we will be going over in the next couple of lessons. If you saved your environment correctly, it should now exist as an option in Postman when you choose your environment. If for any reason your requests still aren't working, double check your work then use resources such as #study-hall, office hours, etc. for further assistance! 
```